internal/api-tests/skr_gcpredisinstance_test.go

```diff
@@ -0,0 +1,107 @@
+package api_tests
+
+import (
+	"github.com/google/uuid"
+	cloudresourcesv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-resources/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+type testGcpRedisInstanceBuilder struct {
```

Comment:
It would be nice this is not just the "test" builder, but is in api

Reply:
Honestly, outside of tests, there wasnt any need for builder for this resource